src/libstore/build.cc

```diff
@@ -2439,7 +2471,7 @@ void DerivationGoal::runChild()
 
             /* Our ancestry. N.B: this uses literal on folders, instead of subpath. Without that,
                you open up the entire filesystem because you end up with (subpath "/") */
-            sandboxProfile += "(allow file-read-metadata\n";
+            sandboxProfile += "(allow file-read*\n";
```

Comment:
Probably fine for now. Let's get this unblocked and then figure out specifically what needs doing here.